web/src/components/pipeline-feed/PipelineFeedItem.vue

```diff
@@ -10,7 +10,7 @@
           <Tooltip>
             <span>{{ since }}</span>
             <template #popper
-              ><span class="font-bold">{{ $t('created') }}</span> {{ created }}</template
+              ><span class="font-bold">{{ $t('repo.pipeline.created') }}</span> {{ created }}</template
```

Comment:
was this always wrong?

Reply:
Well we can at least backport it for v1.0.x